src/etl/LoadBalancer.cpp

```diff
@@ -128,11 +112,18 @@ LoadBalancer::LoadBalancer(
         }
 
         sources_.push_back(std::move(source));
-        LOG(log_.info()) << "Added etl source - " << sources_.back()->toString();
+        chooseForwardingSource();
```

Comment:
This line does nothing. Because the sources all disconnected at this moment . They start to connect at line 124.

Reply:
Yeah, thanks!